qiskit_nature/problems/second_quantization/lattice/lattice_model_problem.py

```diff
@@ -115,4 +115,6 @@ def hopping_qeom_ops(
             Currently, this function is not implemented in the `LtticeProblemModel` class and
             always raises `NotImplementedError`.
         """
-        raise NotImplementedError()
+        raise NotImplementedError(
+            "Currently, it's not implemented in the `LtticeProblemModel` class."
```

Comment:
```suggestion
            "Currently, it's not implemented in the `LatticeProblemModel` class."
```
This copied the spelling error from above in the Returns docstring. I wonder whether we could say something like this problem cannot be used with QEOM solver. @mrossinek Any thoughts, otherwise the text says little beyond whats implied by the error itself.

Reply:
Looking at the code here as well as the qEOM code, it looks to me like this should generally be possible to implement. A quick test could be by testing the behavior of the `ExcitedStatesSolver` + `NumPyEigensolver` (not minimum) to see if that works as expected. If yes, I would handle that accordingly in the result printing and would open a separate issue to track the implementation of the `hopping_qeom_ops` method.

Reply:
I changed `NotImplementedError` to return None. Then, `ExcitedStatesEigensolver` + `NumPyEigensolver` returns some values (I'm not sure whether this behavior is as expected or not).
<img width="509" alt="image" src="https://user-images.githubusercontent.com/47442626/169978856-622f7c06-d0a9-4735-9ec1-10da4388f78a.png">

Reply:
> I changed NotImplementedError to return None. Then, ExcitedStatesEigensolver + NumPyEigensolver returns some values (I'm not sure whether this behavior is as expected or not).

I am not sure  I understand. Are you saying the excited states solver `ExcitedStatesEigensolver + NumPyEigensolver` fails if the `hopping_qeom_ops` raises an error - (and only if it did not, and returned None, did you get those values)? As far as I know its one called by QEOM solver and should never get invoked otherwise.

Reply:
Sorry for the confusion. I thought it would fail and I hadn't check it with `hopping_qeom_ops` + `NotimplementedError`. However, I checked it works, even `hopping_qeom_ops` raises `NotimplementedError`.